servicecontrol/servicecontrol-in-practice.md

```diff
@@ -1,35 +1,51 @@
 ---
 title: ServiceControl in Practice
-summary: Details how to think about the use of ServiceControl in production environments
+summary: Optimizing ServiceControl for use in different environments
 tags:
 - ServiceControl
 ---
 
-ServiceControl is a tool that can accommodate many situations. Features can be enabled or disabled depending on the needs of the environment. Each feature is valuable in certain contexts but also comes with a cost in terms of resource usage and performance. This guidance is provided to provoke thought about why any given feature or plugin might be valuable in a system environment and to help decision makers make informed choices about how to get the best out of ServiceControl without providing solutions to specific scenarios that consider choices around hardware, transport or persistence.
 
-For example, in a development environment the [Debug Session](/servicecontrol/plugins/debug-session.md) and [Saga Audit](/servicecontrol/plugins/saga-audit.md) plugins provide detailed information to support problem analysis. Particular advise that customers not use these plugins outside of a development environment.
+ServiceControl provides many capabilities such as endpoint monitoring, advanced debgugging, and failed message management. These capabilities can be extended by adding optional [plugins](http://docs.particular.net/servicecontrol/plugins/) into the endpoints being monitored. Each capability and plugin provides valuable information, but they have certain resource and performance costs. Harware, peak and average message throughput, and number of endpoints in the system all have a large impact on the performance of ServiceControl. These factors can vary greatly between environments. Capabilities and plugins that provide value in one environment may have a negative impact if included in another environment.
```

Comment:
1. You have a typo. harware -> hardware.
I think you can break this into 2 paragraphs. 

ServiceControl provides many capabilities such as endpoint monitoring, advanced debgugging, and failed message management. These capabilities can be extended by adding optional [plugins](http://docs.particular.net/servicecontrol/plugins/) into the endpoints being monitored. Each capability and plugin provides valuable information, but they have certain resource and performance costs. 

Hardware, peak and average message throughput, and number of endpoints in the system all have a large impact on the performance of ServiceControl. These factors can vary greatly between environments. Capabilities and plugins that provide value in one environment may have a negative impact if included in another environment. For example, the ....

Combine the For example paragraph into the above, so it reads better.